docker/ubuntu/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG ubuntu_image=bionic
+ARG ubuntu_image=22.04
```

Comment:
same here, I expect that multiple steps of the building process would needs to be adapted to 22.04. The best way is to try it on a 22.04 ubuntu vm then update the docker image. I can give it a try

Reply:
I tried it was able to build but you can try to build.

Reply:
did you try by running `./docker-build.sh`? you need to change the image name also in that file as well 
https://github.com/logicalclocks/ePipe/blob/master/docker-build.sh#L20

Reply:
yeah I build it using `docker-build.sh `script

Reply:
I see the `ubuntu_image` argument in the script.  But it appeared to be building from ubuntu22. I will try again.

Reply:
why do we build from `focal ` on arm?

Reply:
I was only using it for testing on mac m1